handbook/strategy/recurring-processes.md

```diff
@@ -0,0 +1,43 @@
+# ⏰ Recurring Processes
+
+This page documents administrative processes that Launch Pad runs every year. In general, both the Leadership team and the Strategy team are responsible for driving efforts on these processes.
+
+## Selecting Leadership
+
+This section documents how Launch Pad's leadership team is selected. Learn more about club leadership in our [manifesto's "club structure" section](/handbook/manifesto.md#club-structure).
+
+### Co-Presidents
+
+Each year (or whenever a co-president steps down), the existing [co-presidents](/handbook/manifesto.md#co-presidents) should nominate their replacements to the current leadership team. Unless any serious objections are raised, the nominees will act as the co-presidents of Launch Pad.
+
+Candidates should be nominated based on active participation in Launch Pad, and understanding of our goals, and an aptitude for mentorship and team management.
+
+There should always be two co-presidents. We should aim to have one technical and one non-technical (i.e. someone from Strategy or Design) co-president when possible.
+
+Relevant handoff tasks are documented in the [leads onboarding guide](/handbook/onboarding/leads.md).
+
+### Leads
+
+Each semester, each [tech lead](/handbook/manifesto.md#project-teams) and the [design lead](/handbook/manifesto.md#design-team) should nominate members they have worked with that they believe would make good leads.
+
+Candidates should be nominated based on active participation in Launch Pad, a robust foundation in relevant technical/design skills, and an aptitude for mentorship. Nominees should be provided with the [tech lead role description](/handbook/recruitment/tech-lead.md) or [design lead role description](/handbook/recruitment/tech-lead.md) to help them decide if they are interested in the role.
+
+There should always be one design lead, and the number of tech leads we need should be evaluated by the leadership team based on club membership.
```

Comment:
```suggestion
There should always be one design lead, and the number of tech leads should be evaluated by the leadership team based on club membership.
```

Also curious about 
> There should always be one design lead

I understand we typically have few designers, which justifies this, but is the "one design lead" expected to stay true for the future?

Reply:
One design lead will probably stay this way - in the past, splitting up the design team seems to have left designers separated, and for the most part the design lead should just help coordinate efforts between design and various parts of the club. For simplicity's sake, at least for the forseeable future, I think it's best to keep it to one - cc @sandyklc in case she has any thoughts on this :)